source/extensions/filters/http/common/jwks_fetcher.cc

```diff
@@ -0,0 +1,91 @@
+#include "extensions/filters/http/common/jwks_fetcher.h"
+
+#include "common/common/enum_to_int.h"
+#include "common/http/headers.h"
+#include "common/http/utility.h"
+
+#include "jwt_verify_lib/status.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Common {
+namespace {
+class JwksFetcherImpl : public JwksFetcher,
+                        public Logger::Loggable<Logger::Id::filter>,
+                        public Http::AsyncClient::Callbacks {
+private:
+  Upstream::ClusterManager& cm_;
+  JwksFetcher::JwksReceiver* receiver_ = nullptr;
+  const ::envoy::api::v2::core::HttpUri* uri_ = nullptr;
```

Comment:
Nit: we tend to just do relative `envoy::api::v2` rather than `::envoy::api::v2`.

Reply:
Fixed in 8fa1dc8